### ISSUE TITLE
Add mediatype and availability status to search results

### ISSUE THREAD

**Issue by caseyedavis12** [WGBH/AAPB2#1084]
We should add mediatype and availability status to the search results. We can remove the titles that appear under each header. This is redundant and takes up space.

**Comment by jasoncorum** [comment]
@caseyedavis12 - For the availability portion, the method that returns access_types provides multiple types for individual results. They are defined in the code as: "all," "online," "on-location," "private," and "digitized." Do you want to provide all access_types defined? Also, do you want to use the values as they're defined in the code? Or would you like something different?

For the media type portion, they are defined as "Moving Image," "Sound", and "other." Are these the values you'd like to display?

**Comment by caseyedavis12** [comment]
@jasoncorum:

For access_types, please show only online and on-location. Please have the label say "Access: Online Reading Room" and "Access: Accessible on location at WGBH and the Library of Congress. [More information](http://americanarchive.org/on-location)


For media type, please show only Moving Image and Sound. If the value is other, please don't display the Media Type.

**Comment by jasoncorum** [comment]
@caseyedavis12 - For some reason I have a vague recollection that we were not going to do this for some reason. Is that correct? Or am I just making that up? :)

**Comment by caseyedavis12** [comment]
@jasoncorum: I think we talked about having this just for screen readers and not visible non-screenreaders. Is that possible? @afred do you remember what we decided on this one?

**Comment by jasoncorum** [comment]
@caseyedavis12 - It is possible and that actually sounds right. I'll move forward with that approach unless @afred remembers otherwise. Thanks!
